java/libphonenumber/pom.xml

```diff
@@ -47,6 +47,14 @@
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.protobuf</pattern>
+                  <shadedPattern>libphonenumber.repackaged.com.google.protobuf</shadedPattern>
```

Comment:
sometime people prepend with com.<company name>, like
```
com.google.libphonenumber.repackaged.com.google.protobuf
```
totally up to you guys 😃